osf/management/commands/project_to_draft_registration_contributor_sync.py

```diff
@@ -0,0 +1,40 @@
+import logging
+
+from django.core.management.base import BaseCommand
+from django.db.models import Count, Q
+
+
+from osf.models import DraftRegistration
+
+logger = logging.getLogger(__name__)
+
+
+def retrieve_draft_registrations_to_sync():
+    # Retrieve all active draft registrations
+    active_draft_registrations = DraftRegistration.objects.filter(Q(deleted__isnull=True) & (Q(registered_node=None) | Q(registered_node__is_deleted=True)))
+    # Retrieve active draft registrations with only 1 contributor (the initiator)
+    active_unsynced_draft_regs = active_draft_registrations.annotate(num_contributor=Count('_contributors')).filter(num_contributor=1)
```

Comment:
add `lte` for num_contributor; check to ensure a draft registration can't end up with no contributors